src/Microsoft.HttpRepl.IntegrationTests/Commands/PrefCommandTests.cs

```diff
@@ -0,0 +1,31 @@
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.HttpRepl.Commands;
+using Microsoft.HttpRepl.IntegrationTests.Mocks;
+using Microsoft.HttpRepl.Preferences;
+using Microsoft.HttpRepl.UserProfile;
+using Microsoft.Repl.Parsing;
+using Xunit;
+
+namespace Microsoft.HttpRepl.IntegrationTests.Commands
+{
+    public class PrefCommandTests
```

Comment:
Should we also test CanHandle(..), GetHelpSummary(..), GetHelpDetails(..)? I added tests and helper here-https://github.com/aspnet/HttpRepl/pull/33/files. Let me know if you can use CommandHelper from there, I'll send out pr with just that file in that case.

Reply:
I added tests for CanHandle and a few for GetHelpDetails. Skipped GetHelpSummary as discussed offline.